src/services/auth/auth.utils.ts

```diff
@@ -11,3 +17,25 @@ export function parseAndValidateRequest(request: unknown, logger: Logger): UserR
   }
   return parsedRequest;
 }
+
+export function parseSub(sub: string): {
+  accountType: AccountType;
+  account: string;
+  network: Network;
+} {
+  const [accountType, account, network] = sub.split('/');
+
+  if (!accountTypes.includes(accountType as AccountType)) {
+    throw new Error(`Invalid account type: ${accountType}`);
+  }
+
+  if (!network.includes(network as Network)) {
+    throw new Error(`Invalid network: ${network}`);
+  }
+
```

Comment:
Doesn't this need username (`account`) validation as well? If not, I would assume that the above checks are just sanity checks.

This also brings up a bigger question. What do usernames look like for DID accounts?

Reply:
I believe they're just ceramic IDs. account is just any network dependant unique identifier in this case

Reply:
What are ceramic IDs? Do you mean `did:key:....`s?

Also, what about the question about validation?

Reply:
yeah they're just sanity checks, they allow us to return them types the type for a username is just string

Reply:
Ok cool. I don't think this should block this PR, but it would probably be better for us to have a better understand of what the `account` portion of the `sub` look like for each network type.